### PR TITLE
Fix crypto-square exercise

### LINKED ISSUE #372: Tests for crypto-square are incorrect
Two of the tests for the `normalize-ciphertext` function of the `crypto-square` exercise are wrong, as they don't account for the padding requirement that is in the README. For example, consider the following test:

```clojure
(deftest cipher-3
  (is (= "msemo aanin dnin ndla etlt shui"
         (crypto-square/normalize-ciphertext "Madness, and then illumination."))))
```

The cipher text that is returned contains chunks of different lengths: `msemo aanin dnin ndla etlt shui`, where the spec says:

> Output the encoded text in chunks that fill perfect rectangles (r X c), with c chunks of r length, separated by spaces. For phrases that are n characters short of the perfect rectangle, pad each of the last n chunks with a single trailing space.

The [spec](https://github.com/exercism/problem-specifications/blob/main/exercises/crypto-square/description.md)'s example string is `"If man was meant to stay on the ground, god would have given us roots."`, and if I run that using the example implementation, it returns `"imtgdvs fearwer mayoogo anouuio ntnnlvt wttddes aohghn sseoau"` instead of `"imtgdvs fearwer mayoogo anouuio ntnnlvt wttddes aohghn  sseoau "`

### PR DESCRIPTION
**Pull request by ErikSchierboom**
The current implementation did not follow the spec. In this commit, the tests are updated as well as the example implementation.

Closes #372

### CODE REVIEW

File: exercises/practice/crypto-square/.meta/src/example.clj [hunk 1]
```diff
@@ -1,37 +1,23 @@
-(ns crypto-square)
+(ns crypto-square, (:require [clojure.string :as str]))
```
Comment (porkostomus) [inline exercises/practice/crypto-square/.meta/src/example.clj:1]: Although insignificant, this comma serves no purpose

Comment (porkostomus) [inline exercises/practice/crypto-square/.meta/src/example.clj:1]: Although insignificant, this comma serves no purpose
```suggestion
(ns crypto-square
  (:require [clojure.string :as str]))
```

File: exercises/practice/crypto-square/.meta/src/example.clj [hunk 2]
```diff
@@ -1,37 +1,23 @@
...
 
-(defn normalize-plaintext [input]
-  (.toLowerCase (clojure.string/replace input #"[^0-9a-zA-Z]" "")))
```
Comment (porkostomus) [inline exercises/practice/crypto-square/.meta/src/example.clj:4]: Whoever did this must have been a Java programmer :p

**Review by porkostomus — APPROVED** [review]

File: exercises/practice/crypto-square/.meta/src/example.clj (hunk 2 above)
Comment (ErikSchierboom) [reply exercises/practice/crypto-square/.meta/src/example.clj:4]: :D

**Pull request merged**